crates/interpreter/src/interpreter.rs

```diff
@@ -68,6 +69,29 @@ impl<EXT: Default, MG: MemoryGetter> Interpreter<EthInterpreter<EXT, MG>> {
             extend: EXT::default(),
         }
     }
+    pub fn with_bytecode(mut self, bytecode: Bytecode) -> Self {
+        self.bytecode = ExtBytecode::new(bytecode);
+        self
+    }
+}
+
+impl Default for Interpreter<EthInterpreter> {
+    fn default() -> Self {
+        Interpreter::<EthInterpreter>::new(
```

Comment:
```suggestion
        Interpreter::new(
```

Probably rust can go without it as it can be deduced from return type

Reply:
It does work. Done